### ISSUE TITLE
unattended_upgrades doesn't work with puppet < 3.5.0 (I think...)

### ISSUE THREAD

**Issue by TravellingGuy** [puppet-community/puppet-unattended_upgrades#13]
puppetmaster version = 3.4.3
Ubuntu 14.04

When trying to use the unattended_upgrades module on my Ubuntu 14.04 server, it kept failing with
> Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Please explicitly
> specify unattended_upgrades::legacy_origin and unattended_upgrades::origins at /etc/puppet
> /environments> /production/modules/unattended_upgrades/manifests/params.pp:46 on node
> blah.blah.com

I compared the puppetlabs apt module with this module to see how it was different and noted this comment in the code:
```
# prior to puppet 3.5.0, defined couldn't test if a variable was defined
# strict variables wasn't added until 3.5.0, so this should be fine.
```
I replaced the $xfacts assignment in the params.pp with the equivalent in the apt module (which does a puppet version check and behaves differently because of strict variables) and it worked.

**Comment by daenney** [comment]
Good point. We fixed that in `apt` but not in `unattended_upgrades`. I'll make sure to do that.

**Comment by apeeters** [comment]
Pull request in #17.

**Comment by igalic** [comment]
fixed in #17
